src/main.py

```diff
@@ -16,11 +16,12 @@
     base_path=BASE_PATH,  # Path of the Directory where the dataset is located.
     test_mode=True,  # Test mode which will only load a subset of the data
     test_rows=33000,  # Number of rows to load in test mode
-    drop_duplicates=True,  # Drop duplicates from the dataset (Currently set true for Copa dataset)
+    drop_duplicates=False,  # Drop duplicates from the dataset (Currently set true for Copa dataset)
```

Comment:
Why did you set this to False?

Reply:
I am still going through this new data, and didn't want it to drop anything right now.